python/ray/cluster_utils.py

```diff
@@ -128,6 +128,16 @@ def remove_node(self, node, allow_graceful=True):
             node (Node): Worker node of which all associated processes
                 will be removed.
         """
+        global_node = ray.worker._global_node
+        if global_node is not None:
+            if node._raylet_socket_name == global_node._raylet_socket_name:
+                ray.shutdown()
```

Comment:
Is the shutdown call here expected?

Reply:
It is not an expected call. I added it to remove misleading error messages that happen due to raylet/GCS crashes.